_posts/2024-02-14-Are-You-Ready-to-Low-Code-Green?-Introduction to-Sustainable-Canvas-App-and-Power-Automate-Development

```diff
@@ -0,0 +1,134 @@
+
+When creating power platform environment from User interface, users cannot select which Datacenter / Gateway region the environment is generated (at least not in Europe Region). You can check the gateway region e.g. by “starting” the new link to data lake.
+
+![A screenshot of power platform page, where user can link to data lake](/img/2024-Power-Platform/PowerApps-RegionInfo.png)
+> Place where I think is easiest to check the accurate region info you environment locates
+
+## The hardware Efficiency
+
+Hardware efficiency considers the efficiency of the data center and end devices, which is mostly beyond the developer's control. The Power Platform operates in Azure, a public cloud, meaning that the level of server utilization is determined by Microsoft. Since most Power Platform projects do not directly control how end devices are managed, the only option where app makers can impact hardware efficiency is by ensuring app efficiency. You want to ensure that users are not tempted to update their end devices simply because the app you created consumes too much memory for their device to handle.
+
+# Carbon aware power platform?
+
+Carbon awareness simply means doing more with our apps when electricity is cleaner and less when it is dirtier (i.e., includes more carbon). At this time, I have not seen instances where there have been such massive data loads that this kind of handling would possibly be worth building or using.
+
+But a girl can dream of a greener future. Could Microsoft provide an eco-mode feature for Power Apps? This would allow end users to decide if they are willing to shift their usage to a cleaner time or reduce the app's performance level. Or could Microsoft provide carbon-aware scheduling for Power Automate, allowing scheduled runs with renewable energy?
+
+[Microsoft is already testing this kind of feature in Windows 11 updates,](https://www.techradar.com/news/windows-11-is-getting-an-eco-friendly-update-but-could-microsoft-do-more) so perhaps someday we will see more carbon awareness in the Power Platform as well.
+
+# Closing words and learning content
+
+"Hopefully, this has provided you with new ideas for finding greener ways to low code. As development progresses towards more sustainable software, we anticipate the emergence of additional best practices and automated solutions to further reduce carbon emissions and energy consumption in Power Platform solutions.
```

Comment:
opening quotation that doesn't seem to end

Reply:
Fixed!